docker-compose.yml

```diff
@@ -223,6 +223,8 @@ services:
       <<: *ccache
       ARROW_ENABLE_TIMING_TESTS:  # inherit
       ARROW_MIMALLOC: "ON"
+      GTest_SOURCE: "BUNDLED"  # Alpine's GTest is not built against
+                               # C++17 so we build from source
```

Comment:
Could you move this to `ci/docker/alpine-linux-3.16-cpp.dockerfile` like other `*_SOURCE=BUNDLED`?

Reply:
Good catch.  I have done this.